### PR TITLE
[CLIENT-7730] Allow Connection specific ICE Servers

### PR DESCRIPTION
**Pull request by mhuynh5757**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [CLIENT-7730](https://issues.corp.twilio.com/browse/CLIENT-7730)

### Description

Introduces a new parameter for `Device.connect` and `Connection.accept` to allow for `Connection` specific ICE Servers.

### Before review
* [x] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [ ] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Squashed erroneous commits if necessary
* [ ] Re-tested if necessary

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+To use this feature, a new parameter `iceServers` can be passed to `Device.connect` and `Connection.accept`. The function signatures are now described as below.
+```ts
+Device.connect(params?: Record<string, string>,
+               audioConstraints?: MediaTrackConstraints | boolean,
+               iceServers?: RTCIceServer[]);
+
+Connection.accept(audioConstraints?: MediaTrackConstraints | boolean,
+                  iceServers?: RTCIceServer[]);
+```
+Passing the `iceServers` parameter to these functions will override any previously set `iceServers` within `Device.options.rtcConfiguration` but not affect any other members set within `Device.options.rtcConfiguration`.
```
Comment (ryan-rowland) [inline CHANGELOG.md:20]: Should we allow all of the members to be changed? What are they exactly?

**Review by ryan-rowland — CHANGES_REQUESTED** [review]
The implementation looks clean. Had one question though.

File: CHANGELOG.md (hunk 1 above)
Comment (mhuynh5757) [reply CHANGELOG.md:20]: The whole typedef is as follows:
```
interface RTCConfiguration {
    bundlePolicy?: RTCBundlePolicy;
    certificates?: RTCCertificate[];
    iceCandidatePoolSize?: number;
    iceServers?: RTCIceServer[];
    iceTransportPolicy?: RTCIceTransportPolicy;
    peerIdentity?: string;
    rtcpMuxPolicy?: RTCRtcpMuxPolicy;
}
```
I was unsure about this one as well considering we pass the whole `RTCConfiguration` to the `PeerConnection`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
-1.13.1 (Jan 11, 2020)
+1.13.2 (In Progress)
```
Comment (ryan-rowland) [inline CHANGELOG.md:1]: New feature should bump this to 1.14.0 yeah?

**Review by ryan-rowland — CHANGES_REQUESTED** [review]
The implementation looks good, I think we just need to bump to 1.14.0 and we should be good to merge.

**Review by ryan-rowland — APPROVED** [review]

**Pull request merged**